core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -72,17 +74,15 @@
  * <li>deleting any topics created internally by Kafka Streams for this application</li>
  * </ol>
  * <p>
- * Do only use this tool if <strong>no</strong> application instance is running.
- * Otherwise, the application will get into an invalid state and crash or produce wrong results.
+ * Do only use this tool if <strong>no</strong> application instance is running. Otherwise, the application will get
+ * into an invalid state and crash or produce wrong results.
```

Comment:
I guess the line break here was inserted on purpose. At least in Streams, we try to break after each sentence in javadocs. It would be odd if the formatter would remove the line break.